nemo/collections/asr/modules/conv_asr.py

```diff
@@ -79,35 +80,18 @@ def input_example(self):
             A tuple of input examples.
         """
         input_example = torch.randn(1, self._feat_in, 8192).to(next(self.parameters()).device)
-        lens = torch.randint(0, input_example.shape[-1], size=(input_example.shape[0],))
-
-        if self._rnnt_export:
-            return tuple([input_example, lens])
-        else:
-            return tuple([input_example])
+        lens = torch.full(size=(input_example.shape[0],), fill_value=8192)
+        return tuple([input_example, lens])
 
     @property
     def disabled_deployment_input_names(self):
         """Implement this method to return a set of input names disabled for export"""
-        if self._rnnt_export:
-            return set([])
-        else:
-            return set(["length"])
+        return set([])
 
     @property
     def disabled_deployment_output_names(self):
         """Implement this method to return a set of output names disabled for export"""
-        if self._rnnt_export:
-            return set([])
-        else:
-            return set(["encoded_lengths"])
-
-    def save_to(self, save_path: str):
-        pass
-
-    @classmethod
-    def restore_from(cls, restore_path: str):
-        pass
+        return set([])
```

Comment:
Better to remove the function altogether ?

Reply:
Yup sounds good.

Reply:
On second thought lets keep it, incase future models require it.